File: csharp/src/JokeApi/Models/Joke.cs
namespace JokeApi.Models;

public class Joke
{
    public int Id { get; set; }
    public string Setup { get; set; } = string.Empty;
    public string Punchline { get; set; } = string.Empty;
    public string Category { get; set; } = string.Empty;
    public string Source { get; set; } = string.Empty;
    public int LolCount { get; set; }
    public int GroanCount { get; set; }
}

File: kotlin/app/src/main/kotlin/com/example/models/Joke.kt
package com.example.models

@kotlinx.serialization.Serializable
data class Joke(
    val id: Int = 0,
    val setup: String = "",
    val punchline: String = "",
    val category: String = "",
    val source: String = "",
    val lolCount: Int = 0,
    val groanCount: Int = 0
)
